src/model/OptionManager.ts

```diff
@@ -108,7 +110,17 @@ class OptionManager {
         // Caution: some series modify option data, if do not clone,
         // it should ensure that the repeat modify correctly
         // (create a new object when modify itself).
-        rawOption = clone(rawOption);
```

Comment:
This clone operation is the one I'm most interested in avoiding as it is extremely expensive in JS heap snapshots